File: employee.c
#include<stdio.h>
#include<stdlib.h>
struct employee
{
	int empno;
	char empname[30];
	char deptname[30];
	int salary;
};
struct employee e[20];
void display(int k);
void main()
{
	int n,i;
	printf("Enter the no.of.employees : ");
	scanf("%d",&n);
	for(i=0;i<n;i++)
	{
		printf("\n\tEmployee %d details :\n",i+1);
		printf("employee number : ");
		scanf("%d",&e[i].empno);
		printf("\nName :");
		scanf("%s",&e[i].empname);
		printf("\nDepartment : ");
		scanf("%s",&e[i].deptname);
		printf("\nSalary : ");
		scanf("%d",&e[i].salary);
	}
	display(n);
}
void display(int k)
{
	printf("\n\n\t\tEmployee Details\n");
	printf("\nEmployeeNumber\tEmployeeName\tDepartmentName\tSalary");
	for(int j=0;j<k;j++)
	{
		printf("\n");
		printf("%d\t\t",e[j].empno);
		printf("%s\t\t",e[j].empname);
		printf("%s\t\t",e[j].deptname);
		printf("%d",e[j].salary);
	}	
}

File: employee.java
import java.util.Scanner;
class employee
{
String emp_name;
int year_of_joining;
long salary;
String address;
public employee(String n,int y,long b,String a)
{
emp_name=n;
year_of_joining = y;
salary=b;
address=a;
}
public static void main(String[] args)
{
employee e1=new employee("david",2002,10000,"chennai");
employee e2=new employee("victor",2003,2000,"chennai");
employee e3=new employee("sam",2005,23455,"mumbai");
System.out.println("\n\tEmployee 1 details : \n");
System.out.println("Name : "+e1.emp_name+"\nYear of joining : "+e1.year_of_joining+"\nSalary : "+e1.salary+"\nAddress : "+e1.address);
System.out.println("\n\tEmployee 2 details : \n");
System.out.println("Name : "+e2.emp_name+"\nYear of joining : "+e2.year_of_joining+"\nSalary : "+e2.salary+"\nAddress : "+e2.address);
System.out.println("\n\tEmployee 3 details : \n");
System.out.println("Name : "+e3.emp_name+"\nYear of joining : "+e3.year_of_joining+"\nSalary : "+e3.salary+"\nAddress : "+e3.address);
}
}
